docs/sources/flow/tasks/configure/configure-kubernetes.md

```diff
@@ -19,36 +19,62 @@ weight: 200
 
 # Configure {{% param "PRODUCT_NAME" %}} on Kubernetes
 
-To configure {{< param "PRODUCT_NAME" >}} on Kubernetes, perform the following steps:
+This page describes how to apply a new configuration to {{< param "PRODUCT_NAME" >}}
+when running on Kubernetes with the Helm chart. It assumes that:
 
-1. Download a local copy of [values.yaml][] for the Helm chart.
+- You have [installed {{< param "PRODUCT_NAME" >}} on Kubernetes using the Helm chart][k8s-install].
+- You already have a new {{< param "PRODUCT_NAME" >}} configuration that you
+  want to apply to your Helm chart installation.
 
-1. Make changes to your copy of `values.yaml` to customize settings for the
-   Helm chart.
+If instead you're looking for help in configuring {{< param "PRODUCT_NAME" >}} to perform a specific task,
+consult the following guides instead:
+- [Collect and forward Prometheus metrics][prometheus],
+- [Collect OpenTelemetry data][otel],
+- or the [tasks section][tasks] for all the remaining configuration guides.
 
-   Refer to the inline documentation in the `values.yaml` for more information about each option.
+[prometheus]: {{< relref "../collect-prometheus-metrics.md" >}}
+[otel]: {{< relref "../collect-opentelemetry-data.md" >}}
+[tasks]: {{< relref "../" >}}
+[k8s-install]: {{< relref "../../get-started/install/kubernetes" >}}
```

Comment:
```suggestion
[prometheus]: ../../collect-prometheus-metrics/
[otel]: ../../collect-opentelemetry-data/
[tasks]: ../
[k8s-install]: ../../../get-started/install/kubernetes/
```

We should stop using relrefs in new doc content. We can use relative URIs for links within the same doc tree.